vscode-extension/package.json

```diff
@@ -0,0 +1,157 @@
+        {
+          "id": "siriusweb-view-container",
+          "title": "Sirius",
+          "icon": "images/icon_32.png"
+        }
+      ]
+    },
+    "views": {
+      "siriusweb-view-container": [
+        {
+          "id": "siriusweb.serversView",
+          "name": "Servers"
+        },
+        {
+          "id": "siriusweb.projectsView",
+          "name": "Projects"
+        },
+        {
+          "id": "siriusweb.explorerView",
+          "name": "Explorer"
+        },
+        {
+          "id": "siriusweb.representationsView",
+          "name": "Representations"
+        }
+      ]
+    },
+    "commands": [
+      {
+        "command": "siriusweb.registerServer",
+        "title": "Register Sirius Server",
+        "category": "Sirius"
+      },
+      {
+        "command": "siriusweb.displayProjects",
+        "title": "Display Projects",
+        "category": "Sirius"
+      },
+      {
+        "command": "siriusweb.displayProjectContents",
+        "title": "Display Project Contents",
+        "category": "Sirius"
+      },
+      {
+        "command": "siriusweb.showRepresentationEditor",
+        "title": "Display Sirius Representation in Editor",
+        "category": "Sirius"
+      }
+    ]
+  },
+  "scripts": {
+    "format": "prettier --write \"src/**/*.{js,ts,tsx,css}\"",
+    "format-lint": "prettier --list-different \"src/**/*.{js,ts,tsx,css}\"",
+    "vscode:prepublish": "npm run compile",
+    "compile": "npm-run-all compile:*",
+    "watch": "npm-run-all -p watch:*",
+    "compile:extension": "tsc -p ./",
+    "compile:views": "webpack  --mode production",
+    "watch:extension": "tsc -watch -p ./",
+    "watch:views": "webpack  --watch --mode production"
+  },
+  "devDependencies": {
+    "@types/mocha": "9.0.0",
+    "@types/node": "16.6.0",
+    "@types/react": "17.0.17",
+    "@types/react-dom": "17.0.9",
+    "@types/react-router-dom": "5.1.8",
+    "@types/uuid": "8.3.1",
+    "@types/vscode": "1.50.0",
+    "@types/websocket": "1.0.4",
+    "css-loader": "6.2.0",
+    "jest-junit-reporter": "1.1.0",
+    "npm-run-all": "4.1.5",
+    "prettier": "2.7.1",
+    "rimraf": "3.0.2",
+    "style-loader": "3.2.1",
+    "ts-loader": "9.3.1",
+    "typescript": "4.7.4",
+    "typescript-formatter": "7.2.2",
+    "webpack": "5.74.0",
+    "webpack-cli": "4.10.0"
+  },
+  "dependencies": {
+    "@apollo/client": "3.6.9",
+    "@babel/core": "7.15.0",
+    "@eclipse-sirius/sirius-components": "2022.9.0",
+    "@eclipse-sirius/sirius-components-charts": "2022.9.0",
+    "@eclipse-sirius/sirius-components-core": "2022.9.0",
+    "@eclipse-sirius/sirius-components-diagrams": "2022.9.0",
+    "@eclipse-sirius/sirius-components-formdescriptioneditors": "2022.9.0",
+    "@eclipse-sirius/sirius-components-forms": "2022.9.0",
+    "@eclipse-sirius/sirius-components-selection": "2022.9.0",
+    "@eclipse-sirius/sirius-components-trees": "2022.9.0",
+    "@eclipse-sirius/sirius-components-validation": "2022.9.0",
```

Comment:
- To update
- Do you really need all that??  I don't see where you should be using validation, formdescriptioneditors etc. It seems that you have copied and paster those dependencies from somewhere (like Sirius Web) while the existing dependencies were way better